compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUTile.cpp

```diff
@@ -24,22 +24,41 @@
 
 namespace mlir {
 namespace iree_compiler {
-namespace {
-
-/// Builds a proper tile sizes vector for the op.
-/// scf::tileUsingSCFForOp expects the num of tile sizes = num of loops. This
-/// method returns a proper tile sizes vector for each op during tiling.
-static SmallVector<OpFoldResult>
-buildTileSizesForOp(OpBuilder &b, Operation *op, ArrayRef<int64_t> tileSizes) {
-  auto tilingOp = cast<TilingInterface>(op);
 
-  SmallVector<int64_t> newTileSizes(tileSizes);
-  newTileSizes.resize(tilingOp.getLoopIteratorTypes().size(), /*default=*/0);
-
-  OpBuilder::InsertionGuard guard(b);
-  return getAsIndexOpFoldResult(b.getContext(), newTileSizes);
+// Note: This is shared with iree-llvmcpu-tile-and-fuse.
+void setSCFTileSizes(scf::SCFTilingOptions &options, TilingInterface consumerOp,
```

Comment:
Can you move the declaration to LLVMCPU/Utils.h and the implementation to Utils.cpp?

Reply:
Done :+1: